src/backend/seq/genpup.py

```diff
@@ -138,6 +131,7 @@ def resized(suffix, b, blklen, last):
     "contig": contig,
     "resized": resized,
 }
+
```

Comment:
Can we move `derived_maps` and maybe `builtin_maps` into `gencomm.py`?

And for `derived_decl_maps`, you could use [getattr](https://docs.python.org/library/functions.html#getattr) to get the function from a string instead of hardcode the map. Example: https://stackoverflow.com/questions/3061/calling-a-function-of-a-module-by-using-its-name-a-string

Reply:
`builtin_maps` are different for seq and cuda.  I'll look into `getattr`.